components/service/experiments/src/test/java/mozilla/components/service/experiments/ExperimentsTest.kt

```diff
@@ -609,32 +712,27 @@ class ExperimentsTest {
             experimentsUpdater.testWaitForUpdate()
         }
 
-        val context = mock(Context::class.java)
-        `when`(context.packageName).thenReturn("test.appId")
-        val sharedPrefs = mock(SharedPreferences::class.java)
-        val prefsEditor = mock(SharedPreferences.Editor::class.java)
-        `when`(prefsEditor.clear()).thenReturn(prefsEditor)
-        `when`(prefsEditor.putBoolean(anyString(), anyBoolean())).thenReturn(prefsEditor)
-        `when`(prefsEditor.putString(anyString(), anyString())).thenReturn(prefsEditor)
-        `when`(sharedPrefs.edit()).thenReturn(prefsEditor)
-        `when`(sharedPrefs.getBoolean(anyString(), anyBoolean())).thenAnswer { invocation -> invocation.arguments[1] as Boolean }
-        `when`(context.getSharedPreferences(anyString(), anyInt())).thenReturn(sharedPrefs)
+        val (
+            context: Context,
+            prefsEditorEnabled: SharedPreferences.Editor,
+            prefsEditorBranch: SharedPreferences.Editor
```

Comment:
nit: since `prefsEditorBranch` isn't being used in this test, you could rename it to `_` to be more explicit about this.